Docker_files/ocsci_container/scripts/run-ocs-ci.sh

```diff
@@ -4,5 +4,15 @@ set -e
 mkdir -p /opt/cluster/logs_ocsci_container
 
 # Execute run-ci
-cd /opt/ocs-ci
-run-ci --cluster-path /opt/cluster "$@" > /opt/cluster/logs_ocsci_container/output.txt
+if [[ "$@" == *"--debug--"* ]]; then
+  cd /opt/ocs-ci-debug
+  pip3.8 install --upgrade pip setuptools
```

Comment:
Think of defining python version of python in common and use for calling this pip or python commands to expose the version from there.

Reply:
done